tests/test_xarray.py

```diff
@@ -122,7 +124,14 @@ def test_open_burst() -> None:
         assert attr_name in res.attrs
         assert res.attrs[attr_name] == COMMON_ATTRIBUTES[attr_name]
     assert res.dims == {"slant_range_time": 21632, "azimuth_time": 1501}
-    assert set(res.variables) == {"VH", "VV", "slant_range_time", "azimuth_time"}
+    assert set(res.variables) == {
```

Comment:
Please, when the expected values spans more than one line use an `expected_` local variable instead.